client/js/main.js

```diff
@@ -46,78 +46,59 @@ function parseInput(e) {
 	e.preventDefault();
 
 	var classifierText = e.currentTarget.querySelector('#classifier-input').value;
-	var jsonText       = e.currentTarget.querySelector('#json-input'      ).value;
+	var jsonText = e.currentTarget.querySelector('#json-input').value;
 
 	if(classifierText.length > 0) {
 		console.log(`DEBUG: parsing classifierText`);
 		// expecting classifierText to be:
-		// http://classifier_url
 		// bodyparam=VALUE
 		// bodyparam=VALUE
 		// bodyparam=VALUE
 		// bodyparam=VALUE
 
-		let classifierUrl = null;
-		let bodyParamsStrings = [];
+		var bodyParamsStrings = [];
 
-		let lines = classifierText.split(/\n/);
+		var lines = classifierText.split(/\n/);
 
-		let mUrl = lines[0].match(/^(https?:.*)/);
-		if( mUrl ) {
-			classifierUrl = mUrl[1];
-			console.log(`DEBUG: parsing classifierUrl=${classifierUrl}`);
-		} else {
-			alert(`no classifier url specified in 1st line`);
-		}
-
-		lines.slice(1).forEach( line => {
-			let mBodyParam = line.match(/^([^=]+)=(.+)$/);
+		Array.from(lines).forEach( function(line) {
+			var mBodyParam = line.match(/^([^=]+)=(.+)$/);
 			if (mBodyParam) {
-				let paramString = `${mBodyParam[1]}=${encodeURIComponent(mBodyParam[2])}`;
+				var paramString = mBodyParam[1] + '=' + encodeURIComponent(mBodyParam[2]);
 				bodyParamsStrings.push( paramString );
-				console.log(`DEBUG: parsing paramString=${paramString}`);
+				console.log('DEBUG: parsing paramString=', paramString);
 			}
 		})
 
-		if(classifierUrl !== null){
-			let bodyText = bodyParamsStrings.join('&');
-			let xhr = new XMLHttpRequest();
-			xhr.open("POST", classifierUrl, true);
-			xhr.setRequestHeader("Content-type", "application/x-www-form-urlencoded");
-			xhr.onreadystatechange = function() {
-				console.log(`classifier response received.`)
-				if(xhr.readyState === XMLHttpRequest.DONE && xhr.status === 200) {
-					try {
-						let jsonText = xhr.response;
-						console.log(`classifier response = ${jsonText}`)
-						let tempResult = JSON.parse(jsonText);
-
-						// display the JSON response from the classifier
-						var jsonDataObj = document.querySelector('.json-data');
-						jsonDataObj.textContent = jsonText;
-
-						data = {results: [{apiResults: tempResult}]};
-						var origin = document.querySelector('.image-container');
-
-						if(tempResult.faces.length > 0) {
-							getImage(tempResult, origin.querySelector('.output'));
-						} else {
-							alert('No faces');
-						}
-					} catch(e) {
-						alert('Invalid JSON');
-						return;
-					}
-				}
-			}
-			console.log(`DEBUG: POST ${classifierUrl}
-${bodyText}`);
-			xhr.send(bodyText);
+		var bodyText = bodyParamsStrings.join('&');
+
+		var serverURL = new URL(window.location.href+'api');
```

Comment:
since this is largely for debugging, and one of the main debugging steps I did was to switch back and forth between the deployed and local instances of the server, it would have been a PITB to have to edit the .env file and restart the tester each time. I agree the default server url should come from the .env file, but I want to be able to change the server request by request via the text box along with the params.